src/main/java/hudson/plugins/templateproject/ProxySCM.java

```diff
@@ -52,19 +47,12 @@ public String getProjectName() {
 		return projectName;
 	}
 
-	public String getExpandedProjectName(AbstractBuild<?, ?> build) {
-		return TemplateUtils.getExpandedProjectName(projectName, build);
-	}
-
-	// Primarily used for polling, not building.
-	public AbstractProject<?, ?> getProject() {
-		return TemplateUtils.getProject(projectName, null);
-	}
-
 	public SCM getProjectScm(AbstractBuild<?, ?> build) {
-		try {
-			return TemplateUtils.getProject(projectName, build).getScm();
-		} catch (Exception e) {
```

Comment:
Exception may mean the runtime exception like NPE? Are you sure you want to delete this catch?